core/domain/stats_domain.py

```diff
@@ -17,16 +17,32 @@
 """Domain object for statistics models."""
 
 import copy
+import logging
 import operator
 import sys
 import utils
 
-from core.domain import exp_services
 from core.domain import interaction_registry
 from core.platform import models
+import feconf
+
 (stats_models,) = models.Registry.import_models([models.NAMES.statistics])
 
 
+# These are special sentinel values attributed to answers migrated from the old
+# answer storage model. Those answers could not have session IDs or time spent
+# values inferred or reconstituted perfectly, so they are assigned these
+# values, instead. Logic and jobs which use these values are expected to skip
+# over the migrated answers to avoid tainted results. Furthermore, all migrated
+# answers are easy to retrieve by reducing session value on this session ID.
+# NOTE TO DEVELOPERS: All other state answer data model entities must not ever
+# store this session ID unless it was created by the AnswerMigrationJob. Also,
+# this string must never change.
+MIGRATED_STATE_ANSWER_SESSION_ID = 'migrated_state_answer_session_id'
```

Comment:
I updated the string itself, but not the variable name. The variable name is easy to rename later on if needed, but I agree with having some sort of time indicator in the string.